docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,187 @@
+---
+sidebar_label: GitHub
+---
+
+# The GitHub Webhook Receiver
+
+The following instructions will work for Github,
+Github Enterprise Cloud, and GitHub Enterprise Server.
```

Comment:
We could probably be more detailed in this section.

It is likely worthwhile to introduce the receiver by describing the events it responds to and what it does in response to them. i.e. A push event refreshes any Warehouses subscribed to the repository from which the event originated. A package event refreshes any Warehouse subscribed to images from a ghcr.io package repository.

Aside: I've just realized the implementation is not yet covering the possibility of a package event coming from a chart repo. I will open a PR for that.